server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -120,4 +126,29 @@ public void stopWithTimeout(long timeout, TimeUnit unit) {
     public int getPort() {
         return ((ServerConnector) jetty.getConnectors()[0]).getLocalPort();
     }
+
+    private static ServerConnector createConnector(Server server, ServerConfig config) {
+        // https://www.eclipse.org/jetty/documentation/jetty-11/programming-guide/index.html#pg-server-http-connector-protocol-http2-tls
+        final HttpConfiguration httpConfig = new HttpConfiguration();
+        final HttpConnectionFactory http11 = new HttpConnectionFactory(httpConfig);
+        final ServerConnector serverConnector;
+        if (config.ssl().isPresent()) {
+            httpConfig.addCustomizer(new SecureRequestCustomizer());
+            final HTTP2ServerConnectionFactory h2 = new HTTP2ServerConnectionFactory(httpConfig);
+            // h2.setRateControlFactory(new RateControl.Factory() {});
+            final ALPNServerConnectionFactory alpn = new ALPNServerConnectionFactory();
+            alpn.setDefaultProtocol(http11.getProtocol());
+            final SslContextFactory.Server sslContextFactory =
+                    (SslContextFactory.Server) JettySSLConfig.forServer(config.ssl().get());
+            final SslConnectionFactory tls = new SslConnectionFactory(sslContextFactory, alpn.getProtocol());
+            serverConnector = new ServerConnector(server, tls, alpn, h2, http11);
+        } else {
+            final HTTP2CServerConnectionFactory h2c = new HTTP2CServerConnectionFactory(httpConfig);
+            // h2c.setRateControlFactory(new RateControl.Factory() {});
```

Comment:
why commented out? for very chatty streams with low amounts of data, this may be required to keep jetty from deciding that the connection is malicious. Note that the comments in the ratecontrol impl is that there is a bug _in other http2 servers_ that current jetty isn't susceptible to, but this is defensive to ensure there isn't a problem, but this might prevent us from keeping some chatty streams going.

Reply:
I was under the impression that this was a placeholder for real rate control in the future? Afaict, this sets up `org.eclipse.jetty.http2.parser.RateControl#NO_RATE_CONTROL`? If it's just a placeholder, I figured commenting out made sense. Regardless, it probably deserves a comment about what we think it's doing, or what we can do with it in the future.

Reply:
Not a placeholder, it is an explicit "you are wrong, don't do this" - I contend that these are broken defaults, not just limits we want to lift.